pkg/cache/placeholder.go

```diff
@@ -73,6 +73,22 @@ func newPlaceholder(placeholderName string, app *Application, taskGroup v1alpha1
 	}
 }
 
+// merge two string maps
+// if the same key defined in the first and second maps
+// the value will be set by the second map
```

Comment:
This function should be defined as part of the `common.utils`. Easier for re-use.

Reply:
Moved to util package for re-use